dipy/stats/resampling.py

```diff
@@ -157,14 +157,22 @@ def __calc_z0(x, p_0, statistic, eps, a_hat, sigma_hat):
         tt_dot[i] = __tt_dot(i, x, p_0, statistic, eps)
     b_hat = b_hat / (2 * n**2)
     c_q_hat = (
-        __tt(x, ((1 - eps) * p_0 + eps * tt_dot / (n**2 * sigma_hat)), statistic)
-        + __tt(x, ((1 - eps) * p_0 - eps * tt_dot / (n**2 * sigma_hat)), statistic)
-        - 2 * __tt(x, p_0, statistic)
+        __tt(
+            x,
+            ((1 - eps) * p_0 + eps * tt_dot / (n**2 * sigma_hat)),
+            statistic=statistic,
+        )
+        + __tt(
+            x,
+            ((1 - eps) * p_0 - eps * tt_dot / (n**2 * sigma_hat)),
+            statistic=statistic,
+        )
+        - 2 * __tt(x, p_0, statistic=statistic)
     ) / eps**2
     return a_hat - (b_hat / sigma_hat - c_q_hat)
 
 
-def __tt(x, p_0, statistic=bs_se):
+def __tt(x, p_0, *, statistic=bs_se):
```

Comment:
There's no decorator here, is that normal?

Reply:
no, needs an update, good catch

Reply:
Okay, Thank you

Reply:
Fixed